lib/Jsrt/Jsrt.cpp

```diff
@@ -5165,14 +5163,23 @@ CHAKRA_API JsRunSerialized(
 {
     PARAM_NOT_NULL(bufferVal);
     const WCHAR *url;
+    JsErrorCode errorCode = ContextAPINoScriptWrapper_NoRecord([&](Js::ScriptContext *scriptContext) -> JsErrorCode {
```

Comment:
Does our usual rule about `{` on new line not apply in lambdas? Just curious, I see some of both in this file so either way can be argued to be consistent with the surrounding style.